### ISSUE TITLE
Is it possible add a feature combo key as input?

### ISSUE THREAD

**Issue by fxliang** [jfedor2/hid-remapper#73]
if yes, it will be a great improvement.

**Comment by jfedor2** [comment]
Can you explain in more detail what you mean by that?

Pressing multiple keys at the same time?

**Comment by fxliang** [comment]
yes, like some qmk keyboard, press multi-keys in the same time (less than a set up duration )  to output a remap key.
e.g.,

 gh  ==> +
 fd   ==> (
 jk    ==> )
 er    ==> {
 ui    ==> }

and so on. 
then the fingers not need to move too long a distance to press keys in the corners.

**Comment by duncanthrax** [comment]
I'd like this too, as discussed elsewhere. One detail I'd like to add is that only the full combo should count as "mapped", individual keys should still count as "unmapped".
